www/js/view_single.js

```diff
@@ -1,45 +1,56 @@
-$(document).ready( function() {
-    $(document).on('change', '.btn-file :file', function() {
-        var input = $(this),
-        label = input.val().replace(/\\/g, '/').replace(/.*\//, '');
-        input.trigger('fileselect', [label]);
-        });
 
-        $('.btn-file :file').on('fileselect', function(event, label) {
-            
-            var input = $(this).parents('.input-group').find(':text'),
-                log = label;
-            
-            if( input.length ) {
-                input.val(log);
-            } else {
-                if( log ) alert(log);
-            }
-        
-    });
-    function readURL(input) {
-        if (input.files && input.files[0]) {
-            var reader = new FileReader();
-            
-            reader.onload = function (e) {
-                $('#img-upload').attr('src', e.target.result);
-            }
-            
-            reader.readAsDataURL(input.files[0]);
-        }
-    }
 
-    $("#imgInp").change(function(){
-        readURL(this);
-    });   
+$(document).ready( function() {
 
     lightbox.option({
       'resizeDuration': 200,
       'wrapAround': true
-    })
+    });
     
 });
 
+Dropzone.autoDiscover = false;
+
+var myDropzone = new Dropzone("#form-upload", {
+    paramName: "file", // The name that will be used to transfer the file
+    maxFilesize: 2, // MB
+    uploadMultiple : true, 
+    parallelUploads: 0,
```

Comment:
Did you play with this variable?

Reply:
ParallelUploads? so many times..  The only thing I actually forgot to test is trying to upload a file that's larger than 2mb.